cvat/apps/engine/models.py

```diff
@@ -418,6 +418,8 @@ class Task(TimestampedModel):
         blank=True, on_delete=models.SET_NULL, related_name='+')
     target_storage = models.ForeignKey('Storage', null=True, default=None,
         blank=True, on_delete=models.SET_NULL, related_name='+')
+    consensus_job_per_segment = models.IntegerField(default=0, blank=True)
+    agreement_score_threshold = models.FloatField(default=0, blank=True)
```

Comment:
Consider extracting this field into a separate model. If we add more parameters in future, it will be easier if it's stored in a dedicated structure.